### ISSUE TITLE
[HP] missing focus state when tabbing through mega menu

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/ibm-dotcom-library#3616]
## Detailed description

> Describe in detail the issue you're having.

There is no focus state appearing when tabbing through the mega menu content. Keyboard navigation works as expected in Storybook.

Recording of keyboard tabbing through mega menu on homepage staging:
![MM- no focus on tab- Aug-18-2020 16-44-51.gif](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/0400c087-8ef8-4e01-8628-73185fcc651e)

> Is this a feature request (new component, new icon), a bug, or a general
> issue?

Bug

> Is this issue related to a specific component?

Mega menu

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?

Chrome
Version 84.0.4147.125 (Official Build) (64-bit)
MacBook pro, osCatalina 10.15.5

> What version of the IBM.com Library are you using?

Latest canary, seen on homepage staging site

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

## Steps to reproduce the issue

1. Open homepage staging
2. Tab to select Products & Solutions
3. Hit enter key to open
4. Tab again to tab into mega menu content

**Comment by RobertaJHahn** [comment]
@oliviaflory This bug is ready to test.  Use the staging link.

**Comment by oliviaflory** [comment]
Looks good @rsebade !

**Comment by RobertaJHahn** [comment]
Also retested by Ram.  Fixed.  Closing.
